src/websocket.c

```diff
@@ -42,6 +42,7 @@
 
 static const uint8_t WS_MASK_SET = 0x80;
 static const uint8_t WS_HEADER_FIN = 0x80;
+static unsigned int fragment_opcode = 0xff;
```

Comment:
This "thing" must not be global, otherwise multiple websocket connections will overwrite this. I think this information should be stored in struct websocket.
Please find a better name for this kind of information.